app/views/static/provider_option_field_input.html.haml

```diff
@@ -1,6 +1,7 @@
 .form-group{"ng-class" => "{{ \"{'has_error': angularForm.\"  + $ctrl.elemID + \".$invalid}\" }}"}
   %label.col-md-2.control-label{"for" => "{{ $ctrl.elemID }}"}
     {{ $ctrl.option.label }}
+    .pficon.pficon-info.fa.fa-info-hint{:title => _("{{ $ctrl.option.help_text }}")}
```

Comment:
This looks like you're enabling 2 different icons `pficon-info` and `fa-info-hint`.

Since `fa-info-hint` is not actually a FontAwesome icon, you probably don't need that `fa`.

And .. can you rename `fa-info-hint` to something that makes it clear this is our color class, not a fontawesome icon class?